src/gui/SettingsWidgetGeneral.ui

```diff
@@ -351,6 +351,58 @@
         </layout>
        </item>
        <item>
+        <layout class="QHBoxLayout" name="autoTypeDelayHorizontalLayout">
+         <property name="spacing">
+          <number>15</number>
+         </property>
+         <property name="sizeConstraint">
+          <enum>QLayout::SetDefaultConstraint</enum>
+         </property>
+         <item alignment="Qt::AlignLeft">
+          <widget class="QLabel" name="autoTypeDelayLabel">
+           <property name="text">
+            <string>Auto-Type delay</string>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <widget class="QSpinBox" name="autoTypeDelaySpinBox">
```

Comment:
Right, the global auto-type shortcut was already hacked together like this so I just continued the tradition. Just to clarify: you want me to wrap both of them in a grid like "Timeouts" is in "Security"?